homeassistant/components/apsystems/config_flow.py

```diff
@@ -48,7 +48,7 @@ async def async_step_user(
                 self._abort_if_unique_id_configured()
                 return self.async_create_entry(
                     title="Solar",
-                    data=user_input,
+                    data={**user_input, MAX_OUTPUT: device_info.maxPower},
```

Comment:
Why is this something stored here and not fetched during runtime?

Reply:
If we would fetch it at runtime when the integration gets loaded and the inverter is offline, it would lead to the integration not starting, right? And since the value won't ever change, I thought it'd be smart to obtain it at setup.

Reply:
I mean if the device is offiline we should not start the integration anyway. Storing it makes it look like its user configurable which its not

Reply:
No, that's wrong. The integration should start as the inverter is offline during the night

Reply:
If a device is offline we should not start as we can't get data etc. That's how it should work

Reply:
Okay, then I'll change it to checking it on startup.